traffic_control/admin/common.py

```diff
@@ -36,3 +36,16 @@ def queryset(self, request, queryset):
         if self.value():
             operational_area = OperationalArea.objects.get(id=self.value())
             return queryset.filter(location__contained=operational_area.location)
+
+
+class ReplacesInline(admin.StackedInline):
+    fk_name = "new"
+    verbose_name = _("Replaces")
+    raw_id_fields = ("old",)
+
```

Comment:
Myös readonly_fields = ("old") ? Jossei tarvi niin sitten joku kommentti, että miksei.

Reply:
Jotta korvausta olisi mahdollista muuttaa administa niin pitää toinen suunta jättää muokattavaksi. Siksi new on readonly. Laitan kommentin.
![image](https://github.com/City-of-Helsinki/city-infrastructure-platform/assets/4151483/404b6c70-9c60-45db-9947-eb0edaed51ec)

Reply:
Jees.. tuosta vielä tiketti varmaan niin ok.